test/recipes/95-test_external_pyca_data/cryptography.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/sh
 #
-# Copyright 2017 The OpenSSL Project Authors. All Rights Reserved.
+# Copyright 2017-2021 The OpenSSL Project Authors. All Rights Reserved.
 # Copyright (c) 2017, Oracle and/or its affiliates.  All rights reserved.
 #
-# Licensed under the OpenSSL license (the "License").  You may not use
+# Licensed under the Apache License 2.0 (the "License").  You may not use
```

Comment:
@mattcaswell please reconfirm

Reply:
> It is my understanding that the CLA of the original author would allow us to relicense their contribution from ASL2.0 back to OpenSSL License, so it should be fine to do this. Correct?

Yes